cookbooks/bcpc-hadoop/recipes/resource_manager.rb

```diff
@@ -107,6 +145,20 @@
   subscribes :restart, "log[jdk-version-changed]", :delayed
 end
 
+service "hadoop-yarn-timelineserver" do
+  action [:enable, :start]
```

Comment:
Don't we need an `only_if { node.roles.include?("BCPC-Hadoop-Head-YarnTimeLineServer") }` around the action to determine if we are going to touch this service or not?